.gitignore

```diff
@@ -1 +1,2 @@
 /target
+Cargo.lock
```

Comment:
Why? I thought it is essential to record changes to this file.

Reply:
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries